FullscreenPopup/FullscreenPopup.js

```diff
@@ -5,6 +5,7 @@
  * @exports FullscreenPopup
  * @exports FullscreenPopupBase
  * @exports FullscreenPopupDecorator
+ * @deprecated Will be removed in 2.0.0. Use {@link agate/Popup} with position="fullscreen" instead.
```

Comment:
Also, would you please add a warning using
`import deprecate from '@enact/core/internal/deprecate';`?
You could refer to the code from `GridListImageItem`.